### PR TITLE
added API calls for whitelabel and image control

### PR DESCRIPTION
**Pull request by dhdanno**
Manipulating the whitelabel doc and images contained therein required some additional functions.

### CODE REVIEW

**Comment by swysor** [comment]
Hey, looking good, though we will need on additional thing to accept this. Can you please add functional tests to the  lib/hz2600/Kazoo/Tests/Functional/ directory? We require these tests be present and passing to accept pull reqeusts for this project. Thank you!

File: lib/hz2600/Kazoo/Api/Collection/Accounts.php [hunk 1]
```diff
@@ -14,6 +17,69 @@ public function fetch(array $filter = array()) {
...
+        $element_wrapper->unfetchable();
+
+        return $this;
+    }
+
+    /**
+     * PUT request to create a new whitelabel doc
+     *
+     */
+    public function whitelabel_create($body) {
```
Comment (swysor) [inline lib/hz2600/Kazoo/Api/Collection/Accounts.php]: For consistency with the project, these function names should all be camelCase, not snake_case.

**Comment by swysor** [comment]
Have not seen any updates on this. We need to make two changes to merge this with the project.
1) function names should follow the convention of lowerCamelCase. 
2) functional tests should be added in https://github.com/2600hz/kazoo-php-sdk/blob/master/docs/UnitTests/index.md

**Comment by dhdanno** [comment]
Hey! Thanks for reminding me. I got busy with other stuff and forgot to come back to this. But I certainly want to get it merged so i'll work on changing the names and adding functional tests.

**Comment by swysor** [comment]
No worries, thanks for helping build this out. Ping us if you need any help with getting the tests working.

**Comment by dhdanno** [comment]
One problem i ran into when running these tests on my dev environment was the rate limiting in crossbar getting triggered due to so many requests being generated. I turned that off in the couch doc and it went away.
I required a sample json object, and i've put that in as a private variable in AccountTest... there is probably a better place for that to go.
And I have not written a test for the image function yet... i'll have to look into how the base64 image needs to get passed in.

**Comment by dhdanno** [comment]
I think there might an issue with the way i implemented the image function. I'm having trouble testing it.

**Comment by swysor** [comment]
Any thing I can help with? What do you mean by "having trouble testing it"?

**Comment by swysor** [comment]
Closing due to lack of updates from submitter. Please re-open when you get a chance to work on this. Thank you!

**Pull request closed without merging**